### ISSUE TITLE
[Tarea 4] AR - demo 

### ISSUE THREAD

**Issue by mmezad** [UC-IIC2613/Syllabus#211]
En el Template de la tarea 4 se muestra un ejemplo del AR, pero no puedo encontrar el código de esta parte (al final de la sección AR - demo), ¿se puede subir ese código o compartir por acá? Gracias de antemano

![pregunta](https://user-images.githubusercontent.com/60619313/126022456-ba62e574-706a-41f3-b9cd-aa979d06bbcb.PNG)

**Comment by jfuentesg26** [comment]
Hola! Acabo de revisar el template de la tarea y ambas funciones que indicas se encuentran definidas. Específicamente el average_episodic_return se encuentra en el recuadro número 11 de código del AR-demo y el animate_agent en el número 14. Espero lo puedas encontrar! Avísame cualquier cosa :)

**Comment by mmezad** [comment]
Sí, esos están. Pero el código de agent no está, solo está el de agent_left y de agent_random. Al iniciar el Template sin editarlo muestra la animación pero no el código donde se crea este agente. 

![pregunta2](https://user-images.githubusercontent.com/60619313/126023191-ea945fa5-c025-4fae-854e-7155fb3ba833.PNG)

Luego al correrlo muestra

![pregunta 3](https://user-images.githubusercontent.com/60619313/126023258-f2e3e51b-6a34-4ff4-a236-9bd9bc81ad82.PNG)

**Comment by jfuentesg26** [comment]
La idea es que el código del agent lo hagas tu, de hecho es la actividad número 6 de la tarea.

**Comment by mmezad** [comment]
ok, no había leído esa parte 😅
